DOL.WHD.Section14c.Api/Controllers/AttachmentController.cs

```diff
@@ -77,14 +78,14 @@ public AttachmentController(IAttachmentService attachmentService, IIdentityServi
         [HttpGet]
         [Route("{EIN}/{fileId}")]
         [AuthorizeClaims(ApplicationClaimTypes.SubmitApplication, ApplicationClaimTypes.ViewAllApplications)]
-        public HttpResponseMessage Download(string EIN, Guid fileId)
+        public IHttpActionResult Download(string EIN, Guid fileId)
         {
             // make sure user has rights to the EIN or has View All Application rights
             var hasEINClaim = _identityService.UserHasEINClaim(User, EIN);
             var hasViewAllFeature = _identityService.UserHasFeatureClaim(User, ApplicationClaimTypes.ViewAllApplications);
             if (!hasEINClaim && !hasViewAllFeature)
             {
-                return new HttpResponseMessage(HttpStatusCode.Unauthorized);
+                return Unauthorized("User don't have rights to the EIN or has View All Application rights"); 
```

Comment:
Wording of this needs changed. Suggestion: "User doesn't have rights to download attachments from this EIN"

Reply:
Will updated the message.

Reply:
@dciceman "User doesn't have rights to download attachments" better from a security perspective. Also EIN related functionality is changing.